tutorials/shading/shading_reference/spatial_shader.rst

```diff
@@ -293,7 +293,7 @@ Below is an example of a custom light function using a Lambertian lighting model
 .. code-block:: glsl
 
     void light() {
-        DIFFUSE_LIGHT += dot(NORMAL, LIGHT) * ATTENUATION * ALBEDO;
+        DIFFUSE_LIGHT += clamp(dot(NORMAL, LIGHT), 0, 1) * ATTENUATION * ALBEDO;
```

Comment:
Please change to 0.0 and 1.0. 

Some GLES2 technically cannot convert from ints to floats implicitly. So the change is needed to ensure this works on all hardware.